website/docs/examples/gps_client/README.md

```diff
@@ -0,0 +1,114 @@
+---
+id: gps-client
+title: GPS service client Example
+---
+
+This [example](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/gps_client/main.py)
+acts as a `GpsClient` in a standalone Python script.
+
+It can process two types of GPS messages: relposned and pvt.
+The user specifies the type of GPS message to parse, and the script reads the corresponding data,
+and prints it to the console.
+
+To run this example, you need a [**farm-ng brain**](/docs/brain/) and a GPS.
+In addition, your PC should be connected to the same network as the brain.
+
+:::info
+If you haven't already cloned the `farm-ng-amiga` repository, do
+so [**here**](/docs/brain/brain-install.md#clone-the-repository).
+:::
+
+### 1. Install the [farm-ng Brain ADK package](/docs/brain/brain-install)
+
+### 2. Install the example's dependencies
+
+:::tip
+
+It is recommended to also install these dependencies and run the
+example in the brain ADK virtual environment.
+
+:::
+
+## Setup
```

Comment:
This should be decreasing in header level since it falls under `2. Install the example's dependencies`
```suggestion
#### Setup
```